src/dotnet/APIView/APIView/Model/TokenTreeModel.cs

```diff
@@ -0,0 +1,182 @@
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+
+namespace csharp_api_parser.TreeToken
```

Comment:
`csharp_api_parser` does not lokk like a correct .NET namespace naming convention. Any reason not to keep this namespace as `ApiView.TreeToken` or simply use namespace `ApiView`?

Reply:
It just used the project name by default

Reply:
Can you please change the project name then? It's not standard project name convention for .NET.